src/fdb5/database/Archiver.h

```diff
@@ -16,7 +16,7 @@
 #ifndef fdb5_Archiver_H
 #define fdb5_Archiver_H
 
-#include <time.h>
+#include <ctime>
```

Comment:
It is unused in the header and only used in the corresponding .cc, should be moved into that.